src/util/embed.py

```diff
@@ -0,0 +1,52 @@
+"""Copyright by CookieDecide, Darkuuu
+Licensed under MIT License
+Contains various helper functions to create embeds for sending. 
+Assures consistent look over different modules.
+"""
+
+import discord
+from discord.ext import commands
+
+
```

Comment:
Again concerning the consistency, in util_audio_player.py we used a seperate class as a wrapper for the implemented methods, so i would recommend to the same style for all.

Reply:
See above